src/components/Table/Table.jsx

```diff
@@ -48,20 +48,36 @@ function Table(props) {
             className={`${borderClasses} ${rowPaddingClass}`}
             role="row"
           >
-            {fields.map((field, index) => (
-              <td
-                data-index={index}
-                className={classNames(borderClasses, {
-                  'vads-u-text-align--left': field.alignLeft,
-                  'medium-screen:vads-u-text-align--right': field.alignRight,
-                })}
-                data-label={field.label}
-                key={`${rowIndex}-${field.label}`}
-                role="cell"
-              >
-                {item[field.value] === null ? '---' : item[field.value]}
-              </td>
-            ))}
+            {fields.map((field, index) =>
+              index === 0 ? (
+                <th
+                  data-index={index}
+                  className={classNames(borderClasses, {
+                    'vads-u-text-align--left': field.alignLeft,
+                    'medium-screen:vads-u-text-align--right': field.alignRight,
+                  })}
+                  data-label={field.label}
+                  key={`${rowIndex}-${field.label}`}
+                  role="cell"
+                  scope="row"
```

Comment:
There's a lot of repetition here.  What do you think about creating an object with the common key/value pairs, and using the spread operator to add to each element? e.g.

```jsx
const cellProps = {
  'data-index': { index },
  className: {
    classNames(borderClasses, {
      'vads-u-text-align--left': field.alignLeft,
      'medium-screen:vads-u-text-align--right': field.alignRight,
    }),
  },
  'data-label': { field.label },
  ...etc.
};

const thProps = {
  ...cellProps,
  scope: 'row',
}


{fields.map((field, index) =>
  index === 0 ? (
    <th {...thProps} />
  ) : (
    <td {...cellProps} />
```

Reply:
Thanks for reviewing! I updated the PR with this approach. I had to use functions to create the objects.